documentation/decisions/adr/2023-12-06-database-migrations.md

```diff
@@ -0,0 +1,111 @@
+# Database Migrations
+
+- **Status:** Active
```

Comment:
```suggestion
- **Status:** Accepted
```

We can write "Accepted" before it is actually finished to make it easier to merge.